github/src/main/java/io/syndesis/github/backend/FileContent.java

```diff
@@ -48,7 +48,7 @@ private String toBase64(byte[] content) {
         if (content == null) {
             return null;
         }
-        return new String(Base64.getEncoder().encode(content));
+        return Base64.getEncoder().encodeToString(content);
```

Comment:
better this way, otherwise one needs to specify charset when constructing String from byte[]